pegasus/sites.v3/code.org/views/contributors.haml

```diff
@@ -0,0 +1,8 @@
+- people.each do |person|
+  .row.film-cast-member
+    %h2.col-xs-12{style: "margin-top: 1em; margin-bottom: .75em; padding-bottom: .5em; border-bottom: 1px solid #b7b7a7;"} #{person[:name_s]}
```

Comment:
mostly because the actual content of this page is likely to change, and I wanted to pollute as few files as possible with unstable code.